lib/layout/dummyLayoutV2.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * dummyLayoutV2.js
+ *
+ * this creates a basic poorly laid out graph
+ *
+ * @author John O'Brien.
+ */
+
+// Create a graph instance.
+
+function buildGraph(graph) {
+  var lib = './../';
```

Comment:
Same as the above for these lines. Double Quotes and move the import statements up to the top level of the file.